data/sumitishere.json

```
@@ -21,28 +21,28 @@
     },
     {
       "icon": "FaLinkedin",
-      "url": "https://www.linkedin.com/in/sumit-mukharjee-2b3769227/"
+      "url": "https://www.linkedin.com/in/sumit-mukharjee-623287280"
     }
   ],
   "links": [
     {
       "group": "Primary Socials",
       "name": "Twitter: Follow me",
-      "url": "https://twitter.com/sumitisheree",
+      "url": "https://twitter.com/sumitverse__",
       "icon": "FaTwitter",
```

[Proofreading] See [my feedback on this line](https://github.com/EddieHubCommunity/LinkFree/pull/7701/files#r1235935089) to see how to improve this one. 
```suggestion
      "icon": "FaTwitter",
```

---

@CBID2 this is incorrect. The property name does not affect if it should end in a comma so I will mark this as resolved.